eng/common/scripts/Update-ChangeLog.ps1

```diff
@@ -108,12 +106,7 @@ if ($LatestsSorted[0] -ne $Version) {
 
 if ($ReplaceLatestEntryTitle)
 {
-    $entryToBeUpdated = $ChangeLogEntries[$LatestVersion]
-    if ($SanitizeEntry)
-    {
-        $entryToBeUpdated = Remove-EmptySections -ChangeLogEntry $entryToBeUpdated
-    }
-    $newChangeLogEntry = New-ChangeLogEntry -Version $Version -Status $ReleaseStatus -InitialAtxHeader $ChangeLogEntries.InitialAtxHeader -Content $entryToBeUpdated
+    $newChangeLogEntry = New-ChangeLogEntry -Version $Version -Status $ReleaseStatus -InitialAtxHeader $ChangeLogEntries.InitialAtxHeader -Content $ChangeLogEntries[$LatestVersion]
```

Comment:
```suggestion
    $newChangeLogEntry = New-ChangeLogEntry -Version $Version -Status $ReleaseStatus -InitialAtxHeader $ChangeLogEntries.InitialAtxHeader -Content $ChangeLogEntries[$LatestVersion].ReleaseContent
```

While working on other stuff I noticed this was broken as you need to pass the content and not the entry here. I'm not sure how this didn't break in any of your testing. Just a basic test of setting the version causes this to give invalid results for me.

Reply:
You can see from the test builds what it looks like without this. https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1094504&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=c9190431-7417-53a5-3fc1-7c7e026cd79d